internal/manifest/manifest.go

```diff
@@ -99,6 +102,12 @@ func (r SNPReferenceValues) Validate() error {
 		return fmt.Errorf("field MicrocodeVersion in manifest cannot be empty")
 	}
 
+	switch r.ProductName {
+	case Milan, Genoa:
```

Comment:
I'd favor a comment here saying that this is meant to not do anything in that case. If we leave it like this, it might look like the case has just been forgotten.